pkg/core/dao/cacheddao.go

```diff
@@ -149,6 +152,66 @@ func (cd *Cached) AppendNEP5Transfer(acc util.Uint160, index uint32, tr *state.N
 	return lg.Size() >= nep5TransferBatchSize, cd.PutNEP5TransferLog(acc, index, lg)
 }
 
+// MigrateNEP5Balances migrates NEP5 balances from old contract to the new one.
+func (cd *Cached) MigrateNEP5Balances(from, to util.Uint160) error {
+	// TODO: deal with caching.
```

Comment:
You have `dropNEP5Cache` flag, so is it still relevant?

Reply:
Good catch, I'll fix that.